src/util/fetchUtils.js

```diff
@@ -137,11 +137,10 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 		credentials: 'same-origin', // allow response to set-cookies
 	};
 	if (hasBody) {
-		config.body = isFormData
-			? queries[0].params
-			: fetchUrl.searchParams.toString();
+		config.body = isFormData ? queries[0].params : searchParams.toString();
 	}
-	const url = isFormData || !hasBody ? fetchUrl.toString() : apiUrl;
+	const url =
+		isFormData || apiUrl + (hasBody ? '' : `?${searchParams.toString()}`);
```

Comment:
yes - if `isFormData` is `true`, then `url` will be set to `true` instead of a URL string.

```js
const apiUrl = '/mu_api';
const hasBody = false;
const searchParams = new URLSearchParams();

// when `isFormData` is `false`, no problem:
let isFormData = false;
console.log(isFormData || apiUrl + (hasBody ? '' : `?${searchParams.toString()}`));
// > /mu_api?

// when `isFormData` is `true`, the assignment short-circuits and returns `true`:
isFormData = true;
console.log(isFormData || apiUrl + (hasBody ? '' : `?${searchParams.toString()}`));
// > true
```